modules/dice/roll-builder.js

```diff
@@ -135,7 +135,7 @@ export default class RollBuilderFFG extends FormApplication {
       }
 
       // validate that required data is present
-      if (this.roll.hasOwnProperty('item') && this.roll.item.hasOwnProperty('flags') && !this.roll.item.flags.starwarsffg.uuid) {
+      if (this.roll.hasOwnProperty('item') && this.roll.item !== undefined && this.roll.item.hasOwnProperty('flags') && !this.roll.item.flags.starwarsffg.uuid) {
```

Comment:
This could still throw an NPE when accessing `starwarsffg.uuid` as `starwarsffg` could be null or undefined.

We'd really be better off getting the actual Item Document and using the Foundry API `getFlag` method, but that is true of an awful lot of places in the old system so this'll do for now.